inc/sai.h

```diff
@@ -135,7 +136,8 @@ typedef enum _sai_api_t
     SAI_API_MACSEC           = 43, /**< sai_macsec_api_t */
     SAI_API_SYSTEM_PORT      = 44, /**< sai_system_port_api_t */
     SAI_API_MY_MAC           = 45, /**< sai_my_mac_api_t */
-    SAI_API_MAX              = 46, /**< total number of APIs */
+    SAI_API_P4EXT            = 46, /**< sai_p4ext_api_t */
+    SAI_API_MAX              = 47, /**< total number of APIs */
```

Comment:
From discussion, this doens't need to be P4 specific,  should it be named generically?

Reply:
Can this be called SAI_API_PROG_INTF (Programmable interface)?